hnn_core/extracellular.py

```diff
@@ -457,11 +457,12 @@ def plot_lfp(self, *, trial_no=None, contact_no=None, tmin=None, tmax=None,
             Trial number(s) to plot
         contact_no : int | list of int | slice
             Electrode contact number(s) to plot
-        tmin : float | None
-            Start time of plot in milliseconds. If None, plot entire
-            simulation.
-        tmax : float | None
-            End time of plot in milliseconds. If None, plot entire simulation.
+        tmin : float | None [deprecated]
+            Start time of plot in milliseconds.
+            If None, plot entire simulation.
```

Comment:
This makes it look like `None` is deprecated. I'd suggest removing the docstring of these arguments altogether since the entire argument will be removed. Also, if we want to move towards deprecating this argument, we don't want to advertise it to the users.